datastream-kafka-connector/src/test/java/com/linkedin/datastream/connectors/kafka/mirrormaker/TestKafkaMirrorMakerConnectorTask.java

```diff
@@ -91,6 +91,7 @@
 /**
  * Tests for {@link KafkaMirrorMakerConnectorTask}
  */
+@Test(singleThreaded = false)
```

Comment:
Should we be doing this in all our test classes? 
I thought by default each test was executed in its own thread and run in parallel. Is that not the case ?